test-run.py

```diff
@@ -163,12 +161,22 @@ def main_loop_consistent(failed_test_ids):
         for task_id in task_ids:
             short_status = worker.run_task(task_id)
             if short_status == 'fail':
+                reproduce_file_path = get_reproduce_file(worker.name)
+                if os.path.exists(reproduce_file_path):
```

Comment:
I propose to wrap this code into a function and place it in 'Utils' section in worker.py in the future follow up commit (where parallel mode will gain support of the reproduce file printing).